docs/pages/config-reference.mdx

```diff
@@ -157,11 +174,15 @@ auth_service:
     authentication:
         # default authentication type. possible values are 'local' and 'github' for OSS
         #  and 'oidc', 'saml' and 'false' for Enterprise.
-        # 'false' is required for FedRAMP / FIPS, see
-        #  https://gravitational.com/teleport/docs/enterprise/ssh-kubernetes-fedramp/
         #  only local authentication (Teleport's own user DB) & Github is supported in the open
         #  source version
         type: local
+
+        # Sets whether local auth is enabled alongside any other authentication
+        # type. Default is true. local_auth must be 'false' for FedRAMP / FIPS.
+        # (https://gravitational.com/teleport/docs/enterprise/ssh-kubernetes-fedramp/)
```

Comment:
`https://goteleort.com/docs`

Reply:
I can go ahead and update the whole file to reflect the new URL.

Reply:
@programmerq can you change this one to point to goteleport.com as well?

Reply:
can do!